gradle-plugin/src/gradleTest/java/com/squareup/anvil/plugin/MoreAsserts.kt

```diff
@@ -0,0 +1,13 @@
+package com.squareup.anvil.plugin
+
+import io.kotest.matchers.file.shouldExist
+import io.kotest.matchers.shouldBe
+import java.io.File
+
+interface MoreAsserts {
+
+  infix fun File.shouldExistWithText(expectedText: String) {
```

Comment:
Is this used in a followup PR? Okay to leave if-so, just making sure it's only temporarily dead code

Reply:
Heh...  oops!

Yeah, it's used in some (currently failing) incremental tests.